src/CONST/index.js

```diff
@@ -216,12 +216,16 @@ const CONST = {
         ACTIONS: {
             LIMIT: 50,
             TYPE: {
-                IOU: 'IOU',
                 ADDCOMMENT: 'ADDCOMMENT',
+                CLOSED: 'CLOSED',
                 CREATED: 'CREATED',
+                IOU: 'IOU',
                 RENAMED: 'RENAMED',
             },
         },
+        ARCHIVE_REASON: {
+            DEFAULT: 'default',
+        },
```

Comment:
This is sort of what I had in mind but I'm wondering if it's worth adding the other reasons also into this constant like accountClosed, accountMerged, removedFromPolicy, policyDeleted as constants even though they're not really used anywhere in the code, it could be good to have for reference?

In the translation files we _could_ do 

```
reportArchiveReasons: {
        [CONST.REPORT.ARCHIVE_REASON.ACCOUNT_CLOSED]: ({login}) => `This workspace chat is no longer active because <strong>${login}</strong> closed their account.`,
        ...
}

to use the const value in the JSON. Very much NAB though.